tools/python/maps_generator/var/etc/map_generator.ini.default

```diff
@@ -1,70 +1,74 @@
-# Do not change it. This is determined automatically.
-# NODE_STORAGE: map
+# Features stage only parallelism level. Set to 0 for auto detection.
+THREADS_COUNT_FEATURES_STAGE: 0
 
 
 [Osm tools]
-# The path to the osmctools sources.
+# Path to osmctools binaries:
+OSM_TOOLS_PATH: ${Developer:OMIM_PATH}/../osmctools
+# If the binaries are not found neither in the configured path nor system-wide,
+# then the tools are built from the sources:
 OSM_TOOLS_SRC_PATH: ${Developer:OMIM_PATH}/tools/osmctools
-# The path where osmctools will be searched or will be built.
-OSM_TOOLS_PATH: ~/osmctools
-
-
-[Stages]
-# Run osmupdate tool for planet.
-NEED_PLANET_UPDATE: 0
-# Auto detection.
-THREADS_COUNT_FEATURES_STAGE: 0
-# If you want to calculate diffs, you need to specify, where old maps are
-DATA_ARCHIVE_DIR: ${Generator tool:USER_RESOURCE_PATH}
-# You may specify, how many versions in the archive to use for diff calculation
-DIFF_VERSION_DEPTH: 2
 
 
 [Logging]
-# The path where maps_generator log will be saved.
-# LOG_FILE_PATH: generation.log
+# maps_generator's general (python output only) log file path and name.
+# More detailed logs that include output of the `generator_tool` binary
+# are located in the `logs/` subdir of a particular build directory,
+# e.g. `maps_build/2023_06_04__20_05_07/logs/`.
+LOG_FILE_PATH: ${Main:MAIN_OUT_PATH}/generation.log
 
 
 [External]
-# Note: If you want to set a directory name you have to add "/" to the end of url.
-# In each field where you need to specify a URL, you can specify the path to the file system using file:///path/to/file
-
-# The url to the planet file.
-# PLANET_URL:
-# The url to the file with md5 sum of the planet.
-# PLANET_MD5_URL:
-# The base url to WorldCoasts.geom and WorldCoasts.rawgeom (without file name).
-# Files latest_coasts.geom and latest_coasts.rawgeom must be at this URL.
+# Planet file location. It should be a dump of OSM data in osm.pbf format,
+# either an entire planet (from "planet.openstreetmap.org") or a particular
+# country/region extract from e.g. [Geofabrik](http://download.geofabrik.de/index.html).
+# Note that an entire planet generation takes 40+ hours on a 256GB RAM server (and 1TB+ disk space).
+# Stick to smaller extracts unless you have a machine this large.
+# Here and further, its possible to specify either an URL (to be downloaded automatically)
+# or a local file path like file:///path/to/file.
+# A sample URL to download a latest OSM dump for Yukon Territories, Canada:
+#PLANET_URL: https://download.geofabrik.de/north-america/canada/yukon-latest.osm.pbf
+#PLANET_URL: file:///home/pastk/dev/OM/osm.pbf/montenegro-latest.osm.pbf
+PLANET_URL: file:///home/pastk/dev/OM/osm.pbf/ile-de-france-latest.osm.pbf
+# Location of the md5 checksum of the planet file:
+#PLANET_MD5_URL: file:///home/pastk/dev/OM/osm.pbf/montenegro-latest.osm.pbf.md5
+PLANET_MD5_URL: file:///home/pastk/dev/OM/osm.pbf/ile-de-france-latest.osm.pbf.md5
```

Comment:
Keep this options empty in default template?

Reply:
Why?
I think its good to have a working default file (e.g. with Yukon, Canada; but maybe better to replace with a smaller land-locked country, so that absence of the coastlines doesn't create visual issues).